core/container/src/main/java/org/wildfly/swarm/Swarm.java

```diff
@@ -740,6 +748,7 @@ public static void main(String... args) throws Exception {
             vme.printStackTrace();
             System.exit(1);
         } catch (final Throwable t) {
+            t.printStackTrace();
             if (System.getProperty("swarm.inhibit.auto-stop") == null) {
                 t.printStackTrace();
```

Comment:
If we do it unconditionally now, two lines above, maybe wanna remove this one?

Reply:
No, it was my debug that I forgot to remove.